python/pyhail/keytable.py

```diff
@@ -79,7 +79,14 @@ def nrows(self):
         
 
     def same(self, other):
-        """Test whether two key-tables are identical
+        """Test whether two KeyTables are identical.
+
+        **Examples**
+
+        >>> kt1 = hc.import_keytable("data/example1.tsv")
+        >>> kt2 = hc.import_keytable("data/example2.tsv")
+        >>> if kt1.same(kt2):
+        >>>     print "KeyTables are the same!"
```

Comment:
We should import print_function from __future__ and use parens:

```# Python 2 and 3:
from __future__ import print_function    # (at top of module)

print('Hello', 'Guido')
```

See: http://python-future.org/compatible_idioms.html